Makefile

```diff
@@ -21,7 +21,7 @@ UV_TICS_GROUPS := "--group=tics"
 include common.mk
 
 .PHONY: format
-format: format-ruff format-codespell format-prettier  ## Run all automatic formatters
+format: format-ruff format-codespell format-prettier format-pre-commit  ## Run all automatic formatters
```

Comment:
pre-commit is also running ruff and prettier. Should they be removed here to not run them twice?

Reply:
It is running them, but I'd rather have a conflict so people know they need to update the versions in their pre-commit config vs. relying on that.